flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/BufferManager.java

```diff
@@ -384,23 +397,20 @@ int unsynchronizedGetFloatingBuffersAvailable() {
         }
 
         /**
-         * Adds an exclusive buffer (back) into the queue and recycles one floating buffer if the
+         * Adds an exclusive buffer (back) into the queue and releases one floating buffer if the
          * number of available buffers in queue is more than the required amount.
```

Comment:
I would add something like that:
> If floating buffer is released, the total amount of available buffers after adding this exclusive buffer has not changed, and no new buffers are available. The caller is responsible for recycling the release/returned floating buffer.

Reply:
Thank you for your reminder, this function description is very necessary